features/support/twopence_init.rb

```diff
@@ -41,22 +47,17 @@
 
 # This function is used to get one of the nodes based on its type
 def get_target(host)
-  case host
-  when 'server'
-    node = $server
-  when 'ceos-minion'
-    node = $ceos_minion
-  when 'ssh-minion'
-    node = $ssh_minion
-  when 'sle-minion'
-    node = $minion
-  when 'sle-client'
-    node = $client
-  when 'sle-migrated-minion'
-    node = $client
-  else
-    raise 'Invalid target.'
-  end
+  map = {
```

Comment:
really  good refactoring! :+1:

Reply:
1. can we avoid to call map ( which is an implicit ruby built-in name) and we can use a more explicit name?

like ```node_hash``` instead of ``` map```

Reply:
OK.